metricbeat/module/kubernetes/util/enrichers.md

```diff
@@ -0,0 +1,51 @@
+## Kubernetes Metadata enrichment
+
+[metadata diag](../_meta/images/enrichers.png)
```

Comment:
I can't see the image in the preview, can you see it @MichaelKatsoulis ?

Reply:
I also don't understand what the image is trying to say... Is it just an example using 3 metricsets on what they are doing?

Reply:
It is supposed to show an example of different metricsets having their own metadata enricher but sharing same watchers which use the k8s API to watch for a resource. 
I am open to all kind of suggestions !!

Reply:
Ok! Maybe you can just write a sentence below the image saying that?